dist/ng-formio-grid.js

```diff
@@ -234,4 +250,5 @@ angular.module('ngFormioGrid', [
Pjwvc3Bhbj4nICsgYnV0dG9uLmxhYmVsICsgJzwvYT4nXG4gICAgICAgICAgICAgIH0pO1xuICAgICAgICAgICAgfSk7XG5cbiAgICAgICAgICAgIHZhciBhZGRDb2x1bW4gPSBmdW5jdGlvbihjb21wb25lbnQsIGNvbHVtbikge1xuICAgICAgICAgICAgICAvLyBFbnN1cmUgdGhhdCB0aGUgbGFiZWxzIGRvIG5vdCBjb2xsaWRlLlxuICAgICAgICAgICAgICB2YXIgbGFiZWw7XG4gICAgICAgICAgICAgIGlmIChjb2x1bW4gJiYgY29sdW1uLmxhYmVsKSB7XG4gICAgICAgICAgICAgICAgbGFiZWwgPSBjb2x1bW4ubGFiZWw7XG4gICAgICAgICAgICAgIH1cbiAgICAgICAgICAgICAgZWxzZSB7XG4gICAgICAgICAgICAgICAgbGFiZWwgPSBjb21wb25lbnQubGFiZWwgfHwgY29tcG9uZW50LmtleTtcbiAgICAgICAgICAgICAgfVxuXG4gICAgICAgICAgICAgIHdoaWxlIChuYW1lcy5oYXNPd25Qcm9wZXJ0eShsYWJlbCkpIHtcbiAgICAgICAgICAgICAgICBsYWJlbCA9IGNvbXBvbmVudC5sYWJlbCArIGluY3JlbWVudCsrO1xuICAgICAgICAgICAgICB9XG5cbiAgICAgICAgICAgICAgbmFtZXNbbGFiZWxdID0gdHJ1ZTtcbiAgICAgICAgICAgICAgJHNjb3BlLmdyaWRPcHRpb25zRGVmLmNvbHVtbkRlZnMucHVzaCh7XG4gICAgICAgICAgICAgICAgY29tcG9uZW50OiBjb21wb25lbnQsXG4gICAgICAgICAgICAgICAgbmFtZTogbGFiZWwsXG4gICAgICAgICAgICAgICAgZmllbGQ6ICRzY29wZS5ncmlkT3B0aW9uc0RlZi5kYXRhUm9vdCArIGNvbXBvbmVudC5rZXksXG4gICAgICAgICAgICAgICAgY2VsbFRlbXBsYXRlOiAnPGRpdiBjbGFzcz1cInVpLWdyaWQtY2VsbC1jb250ZW50c1wiIGJpbmQtaHRtbC1jb21waWxlPVwiQ09MX0ZJRUxEIHwgdGFibGVGaWVsZFZpZXc6dGhpcy5jb2wuY29sRGVmLmNvbXBvbmVudFwiPjwvZGl2PidcbiAgICAgICAgICAgICAgfSk7XG4gICAgICAgICAgICB9O1xuXG4gICAgICAgICAgICBpZiAoJHNjb3BlLmNvbHVtbnMgJiYgJHNjb3BlLmNvbHVtbnMubGVuZ3RoID4gMCkge1xuICAgICAgICAgICAgICB2YXIgY29tcG9uZW50cyA9IEZvcm1pb1V0aWxzLmZsYXR0ZW5Db21wb25lbnRzKGZvcm0uY29tcG9uZW50cyk7XG4gICAgICAgICAgICAgIHZhciBjb2x1bW47XG4gICAgICAgICAgICAgICRzY29wZS5jb2x1bW5zLmZvckVhY2goZnVuY3Rpb24oa2V5KSB7XG4gICAgICAgICAgICAgICAgLy8gUGFzcyBpbiBlaXRoZXIgY29sdW1uIGtleSBvciBjb2x1bW4gZGVmXG4gICAgICAgICAgICAgICAgaWYgKHR5cGVvZiBrZXkgPT09ICdvYmplY3QnKSB7XG4gICAgICAgICAgICAgICAgICBjb2x1bW4gPSBrZXk7XG4gICAgICAgICAgICAgICAgICBrZXkgPSBjb2x1bW4ua2V5O1xuICAgICAgICAgICAgICAgIH1cbiAgICAgICAgICAgICAgICBlbHNlIHtcbiAgICAgICAgICAgICAgICAgIGNvbHVtbiA9IGZhbHNlO1xuICAgICAgICAgICAgICAgIH1cbiAgICAgICAgICAgICAgICBpZiAoY29tcG9uZW50cy5oYXNPd25Qcm9wZXJ0eShrZXkpKSB7XG4gICAgICAgICAgICAgICAgICBhZGRDb2x1bW4oY29tcG9uZW50c1trZXldLCBjb2x1bW4pO1xuICAgICAgICAgICAgICAgIH1cbiAgICAgICAgICAgICAgfSk7XG4gICAgICAgICAgICB9XG4gICAgICAgICAgICBlbHNlIHtcbiAgICAgICAgICAgICAgRm9ybWlvVXRpbHMuZWFjaENvbXBvbmVudChmb3JtLmNvbXBvbmVudHMsIGZ1bmN0aW9uKGNvbXBvbmVudCkge1xuICAgICAgICAgICAgICAgIGlmIChjb21wb25lbnQuaW5wdXQgJiYgY29tcG9uZW50LnRhYmxlVmlldyAmJiBjb21wb25lbnQua2V5KSB7XG4gICAgICAgICAgICAgICAgICBhZGRDb2x1bW4oY29tcG9uZW50KTtcbiAgICAgICAgICAgICAgICB9XG4gICAgICAgICAgICAgIH0pO1xuICAgICAgICAgICAgfVxuXG4gICAgICAgICAgICBnZXRQYWdlKCk7XG4gICAgICAgICAgfSk7XG4gICAgICAgIH07XG5cbiAgICAgICAgJHNjb3BlLiRvbigncmVsb2FkR3JpZCcsIGZ1bmN0aW9uKGV2ZW50LCBzcmMsIHF1ZXJ5KSB7XG4gICAgICAgICAgaWYgKHNyYykge1xuICAgICAgICAgICAgJHNjb3BlLnNyYyA9IHNyYztcbiAgICAgICAgICB9XG4gICAgICAgICAgaWYgKHF1ZXJ5KSB7XG4gICAgICAgICAgICAkc2NvcGUucXVlcnkgPSBxdWVyeTtcbiAgICAgICAgICB9XG4gICAgICAgICAgbG9hZEdyaWQoKTtcbiAgICAgICAgfSk7XG5cbiAgICAgICAgbG9hZEdyaWQoKTtcbiAgICAgIH1cbiAgICBdXG4gIH07XG59KTtcbiJdfQ==
```

Comment:
We need to remove the "debug: true" on the browserify builds for the dist builds. That is pulling in the source maps which we don't want for our distribution.